farmbot_ext/lib/farmbot_ext/amqp/auto_sync_channel.ex

```diff
@@ -191,4 +191,8 @@ defmodule FarmbotExt.AMQP.AutoSyncChannel do
     :ok = Basic.publish(state.chan, @exchange, "bot.#{device}.from_device", json)
     {:noreply, state}
   end
+
+  defp preloader do
+    Application.get_env(:farmbot, :preloader) || Preloader
```

Comment:
We shouldn't be looking up the preloader impl here. The `Preloader` module should do this.